### PR TITLE
added solution

### PR DESCRIPTION
**Pull request by anatolii-zarytskyi**

### CODE REVIEW

File: src/main/java/mate/jdbc/dao/DriverDaoImpl.java [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+            throw new DataProcessingException("Couldn't get driver by id " + id + " ",
+                    throwable);
+        }
+    }
+
+    @Override
+    public List<Driver> getAll() {
+        String query = "SELECT * FROM drivers WHERE deleted = FALSE";
+        try (Connection connection = ConnectionUtil.getConnection();
+                PreparedStatement preparedStatement = connection.prepareStatement(query)) {
```
Comment (ritakisluhina) [inline src/main/java/mate/jdbc/dao/DriverDaoImpl.java:60]: not informative naming

File: src/main/java/mate/jdbc/dao/DriverDaoImpl.java [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+import mate.jdbc.lib.Dao;
+import mate.jdbc.lib.exception.DataProcessingException;
+import mate.jdbc.model.Driver;
+import mate.jdbc.util.ConnectionUtil;
+
+@Dao
+public class DriverDaoImpl implements DriverDao {
+    @Override
+    public Driver create(Driver driver) {
+        String query = "INSERT INTO drivers (name, license) VALUES (?, ?)";
```
Comment (ritakisluhina) [inline src/main/java/mate/jdbc/dao/DriverDaoImpl.java:20]: not informative naming

File: src/main/java/mate/jdbc/model/Driver.java [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+
+import java.util.Objects;
+
+public class Driver {
+    private Long id;
+    private String name;
+    private String licenseNumber;
+
+    public Driver() {
+    }
```
Comment (ritakisluhina) [inline src/main/java/mate/jdbc/model/Driver.java:11]: do you need this?

**Review by ritakisluhina — CHANGES_REQUESTED** [review]
You must also change naming in ManufacturerDaoImpl

**Review by ritakisluhina — APPROVED** [review]

File: src/main/java/mate/jdbc/dao/DriverDaoImpl.java [hunk 4]
```diff
@@ -0,0 +1,112 @@
...
+            throw new DataProcessingException("Couldn't create driver. " + driver + " ",
+                    throwable);
+        }
+    }
+
+    @Override
+    public Optional<Driver> get(Long id) {
+        String getQuery = "SELECT * FROM drivers WHERE id = (?) AND deleted = FALSE";
+        try (Connection connection = ConnectionUtil.getConnection();
+                PreparedStatement getDriver = connection.prepareStatement(getQuery)) {
```
Comment (Max-Pochepets) [inline src/main/java/mate/jdbc/dao/DriverDaoImpl.java:42]:
```suggestion
                PreparedStatement getDriverStatement = connection.prepareStatement(getQuery)) {
```

File: src/main/java/mate/jdbc/dao/DriverDaoImpl.java [hunk 5]
```diff
@@ -0,0 +1,112 @@
...
+import mate.jdbc.lib.Dao;
+import mate.jdbc.lib.exception.DataProcessingException;
+import mate.jdbc.model.Driver;
+import mate.jdbc.util.ConnectionUtil;
+
+@Dao
+public class DriverDaoImpl implements DriverDao {
+    @Override
+    public Driver create(Driver driver) {
+        String createQuery = "INSERT INTO drivers (name, license) VALUES (?, ?)";
```
Comment (Max-Pochepets) [inline src/main/java/mate/jdbc/dao/DriverDaoImpl.java:20]:
```suggestion
        String createQuery = "INSERT INTO drivers (name, license_number) VALUES (?, ?)";
```

**Review by Max-Pochepets — APPROVED** [review]

**Pull request closed without merging**